README.md

```diff
@@ -1,6 +1,6 @@
 [![iRODS](iRODS-Logo.png)](https://docs.irods.org)
 
-The Integrated Rule-Oriented Data System (iRODS) is open source data management software used by research, commercial, and governmental organizations worldwide.
+The Integrated Rule-Oriented Data System (iRODS) is open source, policy-based data management software used by research, commercial, and governmental organizations worldwide.
```

Comment:
The server binary's help text might need to be updated due to this change. I believe I used the words from irods.org.

Reply:
https://github.com/irods/irods/blob/main/server/main_server/src/main.cpp#L394

Reply:
Yep, that's it.

Reply:
thoughts about removing that next sentence from the binary help text?

Reply:
You're talking about this?
```
It virtualizes data storage resources, so users can take control of their data,
regardless of where and on what device the data is stored.
```

Reply:
yes.   happy to leave it.   but not sure it does too much work in that context.

Reply:
i'll remove it.